larq_compute_engine/mlir/tf_tfl_passes.cc

```diff
@@ -211,6 +211,8 @@ void AddTFToLCETFLConversionPasses(
       mlir::TFL::CreateInsertCallOnceOpFromSessionInitializerPass());
 
   pass_manager->addPass(mlir::TFL::CreateLegalizeLCEPass());
+  pass_manager->addPass(mlir::TFL::CreateTranslateToLCEPass());
+  pass_manager->addPass(mlir::TFL::CreateLegalizeLCEPass());
```

Comment:
Was this leftover from testing? I don't think we should do the roundtrip by default on every conversion, or am I missing something?

Reply:
I had added it to ensure that any changes in creating the custom ops will be caught since the tests will still pass even if the custom op creation pass is modified. Now thinking about it, it's better to add an MLIR test that runs both passes together. I'll remove this.